### PR TITLE
Update node js and fix failing unit tests

### PR DESCRIPTION
**Pull request by pieterlukasse**
Related to Jira Ticket: [VADC-59](https://ctds-planx.atlassian.net/browse/VADC-59) , [VADC-247](https://ctds-planx.atlassian.net/browse/VADC-247)

### Bug Fixes
- fixed unhandled promise rejection, which fixes a failing unit test
- fixed failing unit test for UserProfile

### Improvements

- this allows for devs running local node js version > 14 to also get all the tests to pass

### Dependency updates
- updated node js to 16.x in Dockerfile and Travis CI tests

### CODE REVIEW

**Review by mfshao — DISMISSED** [review]
🎉

**Comment by pieterlukasse** [comment]
@mfshao: the labelled tests pass when we update only node js to 16.x and keep npm as is (8.5)

File: src/Submission/MapDataModel.jsx [hunk 1]
```diff
@@ -151,6 +151,9 @@ class MapDataModel extends React.Component {
         if (data && data[this.state.parentNodeType]) {
           this.setState((prevState) => ({ validParentIds: data[prevState.parentNodeType] }));
         }
+      }).catch(error => {
+        // TODO - should we throw this error?
```
Comment (pieterlukasse) [inline src/Submission/MapDataModel.jsx:155]: @mfshao please let me know what you think of this one. Should we throw the error instead?

Comment (mfshao) [reply src/Submission/MapDataModel.jsx:155]: printing is okay for me now

**Review by mfshao — APPROVED** [review]

**Pull request merged**